provisioning/templates-api/kubernetes/templates/etcd/values.yaml

```diff
@@ -10,9 +10,9 @@ podLabels:
 
 podAnnotations:
   log: "true"
-  ad.datadoghq.com/templates-api-etcd.check_names: '["etcd"]'
-  ad.datadoghq.com/templates-api-etcd.init_configs: '[{}]'
-  ad.datadoghq.com/templates-api-etcd.instances: |
+  ad.datadoghq.com/etcd.check_names: '["etcd"]'
+  ad.datadoghq.com/etcd.init_configs: '[{}]'
+  ad.datadoghq.com/etcd.instances: |
```

Comment:
Toto som zle pochopil, ma tam byt `container name`, preto to nefunguje:
![image](https://user-images.githubusercontent.com/19371734/199961537-91b645cc-ee81-4d6c-b0f8-f57c6afd6348.png)

![image](https://user-images.githubusercontent.com/19371734/199961729-6a7d0ffe-78dc-4ec1-bfdf-3ade43d01103.png)